acceptance/tests/direct_puppet/supports_utf8.rb

```diff
@@ -14,17 +11,11 @@
       'audit:refactor',  # Review for agent side UTF validation.
       'server'
```

Comment:
I think the `server` tag will prevent this test from actually executing in puppet component pipelines.

Reply:
Right. Fixed.